sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1952,6 +1952,96 @@ public async Task UpdateSqlServerDataFeedWithEveryMemberAndNewInstance()
+            {
+                Assert.That(dataFeed.Id, Is.Not.Null.And.Not.Empty);
+                Assert.That(dataFeed.Name, Is.Not.Null.And.Not.Empty);
+                Assert.That(dataFeed.Description, Is.Not.Null);
+                Assert.That(dataFeed.Status, Is.Not.Null);
+                Assert.That(dataFeed.Status, Is.Not.EqualTo(default(DataFeedStatus)));
+                Assert.That(dataFeed.AccessMode, Is.Not.Null);
+                Assert.That(dataFeed.AccessMode, Is.Not.EqualTo(default(DataFeedAccessMode)));
+                Assert.That(dataFeed.ActionLinkTemplate, Is.Not.Null);
+                Assert.That(dataFeed.Creator, Is.Not.Null.And.Not.Empty);
+                Assert.That(dataFeed.Administrators, Is.Not.Null);
+                Assert.That(dataFeed.Viewers, Is.Not.Null);
+                Assert.That(dataFeed.IsAdministrator, Is.Not.Null);
+                Assert.That(dataFeed.CreatedTime, Is.Not.Null);
+                Assert.That(dataFeed.CreatedTime, Is.Not.EqualTo(default(DateTimeOffset)));
+
+                Assert.That(dataFeed.MissingDataPointFillSettings, Is.Not.Null);
+                Assert.That(dataFeed.MissingDataPointFillSettings.FillType, Is.Not.Null);
+                Assert.That(dataFeed.MissingDataPointFillSettings.FillType, Is.Not.EqualTo(default(DataFeedMissingDataPointFillType)));
+
+                if (dataFeed.MissingDataPointFillSettings.FillType == DataFeedMissingDataPointFillType.CustomValue)
+                {
+                    Assert.That(dataFeed.MissingDataPointFillSettings.CustomFillValue, Is.Not.Null);
+                }
+                else
+                {
+                    Assert.That(dataFeed.MissingDataPointFillSettings.CustomFillValue, Is.Null);
+                }
+
+                Assert.That(dataFeed.Granularity, Is.Not.Null);
+                Assert.That(dataFeed.Granularity.GranularityType, Is.Not.EqualTo(default(DataFeedGranularityType)));
+
+                if (dataFeed.Granularity.GranularityType == DataFeedGranularityType.Custom)
+                {
+                    Assert.That(dataFeed.Granularity.CustomGranularityValue, Is.Not.Null);
+                }
+                else
+                {
+                    Assert.That(dataFeed.Granularity.CustomGranularityValue, Is.Null);
+                }
+
+                Assert.That(dataFeed.Schema, Is.Not.Null);
+                Assert.That(dataFeed.Schema.MetricColumns, Is.Not.Null);
+
+                foreach (DataFeedMetric metric in dataFeed.Schema.MetricColumns)
+                {
+                    Assert.That(metric, Is.Not.Null);
+                    Assert.That(metric.MetricId, Is.Not.Null.And.Not.Empty);
+                    Assert.That(metric.MetricName, Is.Not.Null.And.Not.Empty);
+                    Assert.That(metric.MetricDisplayName, Is.Not.Null.And.Not.Empty);
+                    Assert.That(metric.MetricDescription, Is.Not.Null);
```

Comment:
does it always contain a value?

Reply:
Yes. Server makes descriptions empty when not specified during creation.